UnityProject/Assets/Scripts/UI/Systems/AdminTools/AdminGlobalAudio.cs

```diff
@@ -12,35 +17,116 @@ namespace AdminTools
 	public class AdminGlobalAudio : MonoBehaviour
 	{
 		[SerializeField] private GameObject buttonTemplate = null;
+		[SerializeField] private Scrollbar loadingBar = null;
+		[SerializeField] private GameObject loadingView = null;
 		private AdminGlobalAudioSearchBar SearchBar;
 		public List<GameObject> audioButtons = new List<GameObject>();
 
 		[SerializeField] private AudioClipsArray audioAddressables = null;
 
-		public List<AddressableAudioSource> audioList;
+		public Dictionary<AddressableAudioSource, string> audioList = new Dictionary<AddressableAudioSource, string>();
+		public CatalogueData catalogueData;
 
 		private void Awake()
 		{
 			SearchBar = GetComponentInChildren<AdminGlobalAudioSearchBar>();
-			
-			DoLoadAudio(audioAddressables);
+#if UNITY_EDITOR
+			catalogueData = AssetDatabase.LoadAssetAtPath<CatalogueData>("Assets/CachedData/CatalogueData.asset");
+#endif
+
+			Refresh();
+			DoLoadAudio(catalogueData);
+		}
+
+		private static List<string> GetCataloguePath()
+		{
+#if UNITY_EDITOR
+			var path = Application.dataPath.Remove(Application.dataPath.IndexOf("/Assets", StringComparison.Ordinal));
+			path = Path.Combine(path, "AddressablePackingProjects");
+#else
+			var path = Path.Combine(Application.streamingAssetsPath, "AddressableCatalogues");
```

Comment:
it might be better to, get it from the bundle is loaded instead of the local bundles, so if someone has a custom bundle it will automatically update

Reply:
no idea how to do that